llvm/lib/ProfileData/Coverage/CoverageMapping.cpp

```diff
@@ -458,7 +395,7 @@ class MCDCRecordProcessor {
     MCDCRecord::TestVector TV(NumConditions, MCDCRecord::MCDC_DontCare);
 
     // Use the base test vector to build the list of all possible test vectors.
-    buildTestVector(TV);
+    buildTestVector(TV, 1, 0);
```

Comment:
Minor nit: I wonder if it would be of some value to include a very brief comment that explains what the initial values mean? Up to you.